### DISCUSSION TITLE
A stylish unicorn mascot?

### DISCUSSION THREAD

**Discussion by asamiller** [jpudysz/react-native-unistyles#43] [Show and tell]
This library looks great! I ask DALLE3 to make a stylish unicorn. Feel free to use it if you want.

![DALL·E 2023-10-18 15 37 36 - 3D render of a stylish unicorn in a charcoal gray suit with a matching tie, posing confidently on a glittering hill, its vibrant mane reflecting the c](https://github.com/jpudysz/react-native-unistyles/assets/5133623/37e03fd6-af58-4bcf-be9f-70409dee6a87)

![DALL·E 2023-10-18 15 37 41 - 3D render of a stylish unicorn wearing a tailored navy blue suit, standing elegantly on a glittering hill, its shimmering mane flowing in the gentle b](https://github.com/jpudysz/react-native-unistyles/assets/5133623/95c668f4-3926-4198-9e15-51281d28bd5f)

![DALL·E 2023-10-18 15 35 17 - 3D render of a stylish unicorn wearing a sophisticated suit, standing majestically on a glittering hill, its mane flowing in the wind, showcasing vibr](https://github.com/jpudysz/react-native-unistyles/assets/5133623/ec1cb9a5-1335-4de1-9bac-fc3181b9328e)

**Comment by jpudysz** [comment]
Thanks @asamiller , for now I want to keep it simple 😅
